src/style.css

```
@@ -0,0 +1,105 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: heavy;
+  font-weight: 800;
+}
+
+body {
+  font-family: "Avenir", sans-serif;
+  font-weight: 300;
+  color: #3d4e61;
+}
+
+.searchBarBig {
+  width: 100%;
+  margin-top: 20px;
+
+  position: relative;
+}
+
+.searchBarSmall {
+  width: 100%;
+
+  position: relative;
+}
+
+.form {
+  margin-bottom: 20px;
+}
```

you only need `form` selector for that
```suggestion
.form {
  margin-top: 20px;

  position: relative;
}
```